roles/kubernetes-apps/metrics_server/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+metrics_server_namespace: "kube-system"
+metrics_server_kubelet_insecure_tls: true
+metrics_server_kubelet_preferred_address_types: "InternalIP"
```

Comment:
Can this be multiple entries like the `kubelet_preferred_address_types: 'InternalDNS,InternalIP,Hostname,ExternalDNS,ExternalIP'` ? Because then I would suggest to set it equally